---
title: MSSQLSERVER_26013 | Microsoft-Dokumentation
ms.custom: ''
ms.date: 03/06/2017
ms.prod: sql-server-2014
ms.reviewer: ''
ms.suite: ''
ms.technology: supportability
ms.tgt_pltfrm: ''
ms.topic: conceptual
helpviewer_keywords:
- 26013 (Database Engine error)
ms.assetid: 7362fca4-b005-4872-bdfa-a3e32f834db8
caps.latest.revision: 9
author: MashaMSFT
ms.author: mathoma
manager: craigg
ms.openlocfilehash: 3ec804b657be554572b4557c9d26918220f1b2a6
ms.sourcegitcommit: f8ce92a2f935616339965d140e00298b1f8355d7
ms.translationtype: MT
ms.contentlocale: de-DE
ms.lasthandoff: 07/03/2018
ms.locfileid: "37422919"
---
# <a name="mssqlserver26013"></a>MSSQLSERVER_26013
    
## <a name="details"></a>Details  
  
|||  
|-|-|  
|Produktname|SQL Server|  
|Ereignis-ID|26013|  
|Ereignisquelle|MSSQLSERVER|  
|Komponente|SQLEngine|  
|Symbolischer Name|SNI_SSL_READY|  
|Meldungstext|Das Zertifikat [Cert Hash(sha1) "%hs"] wurde für die Verschlüsselung erfolgreich geladen.|  
  
## <a name="explanation"></a>Erklärung  
 Diese Meldung dient zu Informationszwecken. Das genannte Zertifikat wurde erfolgreich in [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] geladen und kann für die Verschlüsselung verwendet.  
  
## <a name="user-action"></a>Benutzeraktion  
 Es ist keine Aktion erforderlich.  
  
  
